src/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -45,6 +45,15 @@ public static Process[] GetProcessesByName(string processName, string machineNam
             return list.ToArray();
         }
 
+        public int HandleCount
+        {
+            get
+            {
+                EnsureState(State.HaveProcessInfo);
+                return _processInfo.HandleCount;
+            }
+        }
```

Comment:
I think caching this for Unix would be a good thing to do, especially since we already have a place to cache it: ProcessInfo.HandleCount.  Consider changing this up a bit, e.g. have HandleCount in the main partial file that does:
```C#
public int HandleCount
{
    get
    {
        EnsureState(State.HaveProcessInfo);
        EnsureHandleCountPopulated();
        return _processInfo.HandleCount;
    }
}

partial void EnsureHandleCountPopulated();
```
Don't touch the Windows file, and in the Unix files do:
```C#
partial void EnsureHandleCountPopulated()
{
    if (_processInfo.HandleCount <= 0)
    {
        _processInfo.HandleCount = ...; // the code you currently have
    }
}
```
or something like that.